docs: clarify `fetch` request de-duplication (#52100)
vercel/next.js: docs/02-app/01-building-your-application/03-data-fetching/index.mdx

> **Good to know**:
>

> > - Fetch requests are not automatically deduplicated under some of the conditions. [Learn more about caching](/docs/app/building-your-application/data-fetching/caching).

Reviewer: ```suggestion
> - `fetch` requests are automatically deduplicated [under the following conditions](/docs/app/building-your-application/data-fetching/caching).
```
